### PR TITLE
Update to native SDKs

### PR DESCRIPTION
**Pull request by Cloudwu1**
* [X] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,27 @@ This preview version adds support for .NET Core 3.0 WPF and WinForms application
...
+
+* **[Fix]** Fix confusing information log about the availability of the Firebase SDK.
+* **[Fix]** Fix sending the push installation log after delayed start.
+
+### App Center Auth
+
+#### iOS and Android
+
+* **[Feature]** App Center Auth logging now includes MSAL logs.
+* **[Fix]** Redirect URIs are now hidden in logs.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: iOS PR isn't a part of this release https://github.com/microsoft/appcenter-sdk-apple/pull/1727

Comment (Cloudwu1) [reply CHANGELOG.md]: Delete in new commit. Please review again.

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +31,30 @@ This preview version adds support for .NET Core 3.0 WPF and WinForms application
...
+* **[Fix]** Obfuscate app secret value that appears as URI part in verbose logs for in-app updates.
+
+### App Center Push
+
+#### Android
+
+* **[Fix]** Fix confusing information log about the availability of the Firebase SDK.
+* **[Fix]** Fix sending the push installation log after delayed start.
+
+### App Center Auth
```
Comment (MatkovIvan) [inline CHANGELOG.md]: Nitpick: we usually sort the services

Comment (Cloudwu1) [reply CHANGELOG.md]: Updated. @MatkovIvan

Comment (MatkovIvan) [reply CHANGELOG.md]: I meant place Auth before Distribute and Push.
Anyway, not big deal

**Pull request merged**